assets/css/vars.css

```diff
@@ -12,6 +12,9 @@
     --header-menu-top-gap: 1rem;
 
     /* Typography */
+    --font-body: 'Literata', Georgia, Cambria, 'Noto Serif', 'Droif Serif', ui-serif, serif;
```

Comment:
@tomfran this should be a more reasonable fallback.

Reply:
I'll have a look at soon as I have a laptop on my hands, thank you for changing it :)